test/runtests.jl

```diff
@@ -20,8 +20,6 @@ if GROUP == "Core" || GROUP == "All"
     @time @safetestset "Hessian colorvecs" begin include("test_sparse_hessian.jl") end
     @time @safetestset "Integration test" begin include("test_integration.jl") end
     @time @safetestset "Special matrices" begin include("test_specialmatrices.jl") end
-    @time @safetestset "Jac Vecs and Hes Vecs" begin include("test_jaches_products.jl") end
-    @time @safetestset "Vec Jac Products" begin include("test_vecjac_products.jl") end
```

Comment:
these tests were copied to `InterfaceI` but never removed from `All`.

https://github.com/JuliaDiff/SparseDiffTools.jl/blob/65dbe8d6271bf4d6ee4c7a9e81c469b54cf2e203/test/runtests.jl#L26-L29